static/src/stylesheets/amp/_hosted.scss

```diff
@@ -334,6 +334,10 @@ $badge-width: $gs-column-width + $gs-gutter;
     bottom: 0;
     width: $badge-width;
     z-index: $zindex-ads;
+    margin: 0 $gs-gutter/2;
```

Comment:
I tried it, but I need a margin rather than a padding, as there is a background colour on the badge. Is there an equivalent with margins instead?